server/tso/tso.go

```diff
@@ -39,6 +39,8 @@ const (
 	// When a TSO's logical time reaches this limit,
 	// the physical time will be forced to increase.
 	maxLogical = int64(1 << 18)
+	// SuffixBits indicates the number of suffix bits
+	SuffixBits = 8
```

Comment:
we already meet the problem about:
![image](https://user-images.githubusercontent.com/6428910/101461675-3307b780-3976-11eb-98e6-bab6a0bc669e.png)
assume we need 1000,000 QPS, that's mean 1000 ts need to be allocated on average every millisecond, the buffer of logical part here is obviously not enough

Reply:
Making the bits of suffix change dynamicly may alleviates this problem. but can not to solve it. Do you have any idea?

Reply:
we can make it be dynamic, but we also limit the upper bits.

Reply:
I have already made it be dynamic and also limit the upper to the 2^8 now. PTAL.